libs/core/src/lib/message-strip/message-strip.component.html

```diff
@@ -1,3 +1,13 @@
+<span class="fd-accessible-hidden-message" [attr.aria-label]="_getMessage()"></span>
+<span
+    *ngIf="_customAriaLabel"
+    class="fd-accessible-hidden-message"
+    [id]="_customAriaLabelId"
+    [attr.aria-label]="_customAriaLabel"
+></span>
+<p class="fd-message-strip__text" [id]="_messageContentId" aria-hidden="true" role="alert">
```

Comment:
after removing the invisible element and moving its aria-label to the host, I think we can get rid of this aria-hidden="true"